easybuild/easyconfigs/t/Transformers/Transformers-4.2.1-foss-2020a-Python-3.8.2.eb

```diff
@@ -0,0 +1,43 @@
+easyblock = 'PythonBundle'
+
+name = 'Transformers'
+version = '4.2.1'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/facebookresearch/detectron://github.com/huggingface/transformers'
```

Comment:
@smoors the `homepage` URL looks wrong - pasting error?

Reply:
indeed, fixed in https://github.com/easybuilders/easybuild-easyconfigs/pull/12032/commits/3aa1fd85d53d997540751e1e07b500849cb001f0